prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR_to_HL7/elims_2_40_05059364_34872_MIN-expected.hl7

```diff
@@ -3,6 +3,6 @@ SFT|CDC^^^^^CDC&2.16.840.1.114222.4&ISO^XX^^^CDC CLIA|ELIMS V11|STARLIMS|Binary
 NTE|1|L|SPHL Submitter: PR Public Health Lab, Submitter ID: SPHL-000049, Address: Periferal St. Bldg. A, 2nd Fl. Call Box 70184 San Juan, Puerto Rico 00936 United States, Email: PRDHPHL@salud.gov.pr, Submitter Patient ID: , Submitter Alt Patient ID: , Submitter Specimen ID: , Submitter Alt Specimen ID:|RE^Remark^HL70364^^^^2.5.1^^^^^^^2.16.840.1.113883.12.364
 ORC|RE|CDC_3015960902^CDC^1.111.122311.222.44.2.3.3^ISO|40_3015960902_05059364_1087^STARLIMS.CDC.Stag^2.16.840.1.114222.4.3.3.2.1.2^ISO|||||||||SPHL-000049^PR Public Health Lab^^^^^^^STARLIMS.CDC.Stag&2.16.840.1.114222.4.3.3.2.1.2&ISO^^^^XX||^NET^Internet^PRDHPHL@salud.gov.pr|||||||PR Public Health Lab^D^^^^STARLIMS.CDC.Stag&2.16.840.1.114222.4.3.3.2.1.2&ISO^XX^^^SPHL-000049|Periferal St. Bldg. A, 2nd Fl.^Call Box 70184^San Juan^PR^00936^USA^M|^WPN^Internet^PRDHPHL@salud.gov.pr|Periferal St. Bldg. A, 2nd Fl.^Call Box 70184^San Juan^PR^00936^USA^M
-OBR|1|CDC_3015960902^CDC^1.111.122311.222.44.2.3.3^ISO|40_3015960902_05059364_1087^STARLIMS.CDC.Stag^2.16.840.1.114222.4.3.3.2.1.2^ISO|PLT1228^Mold and Yeast XXX MS.MALDI-TOF^PLT^1087^MALDI-TOF-CLIA^L^2.74^v unknown^^^^^^2.16.840.1.113883.6.1||||||||||||SPHL-000049^PR Public Health Lab^^^^^^^STARLIMS.CDC.Stag&2.16.840.1.114222.4.3.3.2.1.2&ISO^^^^XX|^NET^Internet^PRDHPHL@salud.gov.pr|||||202308231924-0400|||F
-OBX|1|CWE|PLT1228^Mold and Yeast XXX MS.MALDI-TOF^PLT^3562^MALDI-TOF-CLIA^L^2.74^v_unknown^MALDI-TOF-CLIA|ZZYGQ1OS-1|53326005^Candida albicans (organism)^SCT^^^^09012018^^Candida albicans||||||F||||11D0668319^Centers for Disease Control and Prevention^CLIA^40^Fungus Reference Laboratory^L|NXQ0@cdc.gov^Anderson^Christopher|||20230823191530-0400||||Centers for Disease Control and Prevention^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^11D0668319|1600 Clifton Rd^^Atlanta^GA^30329^USA^B
-SPM|1|^3015960902&STARLIMS.CDC.Stag&2.16.840.1.114222.4.3.3.2.1.2&ISO||119297000^Blood specimen^SCT^BLD^Blood^L^0912017^Adobe_Code^Blood||||||||||,||||20230823181001+0000
\ No newline at end of file
+OBR|1|CDC_3015960902^CDC^1.111.122311.222.44.2.3.3^ISO|40_3015960902_05059364_1087^STARLIMS.CDC.Stag^2.16.840.1.114222.4.3.3.2.1.2^ISO|PLT1228^Mold and Yeast XXX MS.MALDI-TOF^PLT^1087^MALDI-TOF-CLIA^L^2.74^v unknown^^^^^^2.16.840.1.113883.6.1|||0000|||||||||SPHL-000049^PR Public Health Lab^^^^^^^STARLIMS.CDC.Stag&2.16.840.1.114222.4.3.3.2.1.2&ISO^^^^XX|^NET^Internet^PRDHPHL@salud.gov.pr|||||202308231924-0400|||F
+OBX|1|CWE|PLT1228^Mold and Yeast XXX MS.MALDI-TOF^PLT^3562^MALDI-TOF-CLIA^L^2.74^v_unknown^MALDI-TOF-CLIA|ZZYGQ1OS-1|53326005^Candida albicans (organism)^SCT^^^^09012018^^Candida albicans||||||F|||0000|11D0668319^Centers for Disease Control and Prevention^CLIA^40^Fungus Reference Laboratory^L|NXQ0@cdc.gov^Anderson^Christopher|||20230823191530-0400||||Centers for Disease Control and Prevention^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^11D0668319|1600 Clifton Rd^^Atlanta^GA^30329^USA^B
```

Comment:
So regarding the 0000 value, it seems like the only way we would have 0000 here is if that was the original value of the HL7v2 that was sent to us? What if the sender sent FHIR with a missing date, what would happen? According to HL7v2 spec, we should put `0000`.

Reply:
I will try to find where this is documented and create a new ticket for this work.